install_config/configuring_rhv.adoc

```diff
@@ -11,20 +11,8 @@
 
 toc::[]
 
-== Configuring for Red Hat Virtualization
-
 You can configure {product-title} to use Red Hat Virtualization.
```

Comment:
This sentence is floating between the TOC and "Configuring Red Hat Virtualization objects". Is that intentional?

Reply:
Yes, the [assembly reference](https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_ASSEMBLY_a-collection-of-modules.adoc) has a short description floating between the H1 and the prereqs section (or includes). 

However, the reference doesn't have a TOC callout. This might be one more thing to decide in the 4.0 standards guideline.